code/__DEFINES/dcs/signals.dm

```diff
@@ -1010,3 +1010,6 @@
 
 /// from /obj/structure/cursed_slot_machine/determine_victor() when someone finally wins.
 #define COMSIG_GLOB_CURSED_SLOT_MACHINE_WON "cursed_slot_machine_won"
+
+///from of mob/MouseDrop(): (/atom/over, /mob/user)
+#define COMSIG_DO_MOB_STRIP "cursed_slot_machine_won"
```

Comment:
Is it supposed to be "cursed_slot_machine_won" for COMSIG_DO_MOB_STRIP?

Reply:
uhhh, no I just forgot